contentcuration/contentcuration/dev_settings.py

```diff
@@ -33,5 +33,6 @@
     'debug_toolbar.panels.signals.SignalsPanel',
     'debug_toolbar.panels.logging.LoggingPanel',
     'debug_toolbar.panels.redirects.RedirectsPanel',
+    # 'pympler.panels.MemoryPanel',
```

Comment:
There was a problem with rendering certain pages, so I left that in in case someone wants to use that functionality for specific pages